components/sensor/teleinfo.rst

```diff
@@ -97,6 +97,34 @@ Configuration variables:
 - **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`UART Component <uart>` if you want
   to use multiple UART buses.
 
+
+.. warning::
+
+    UART hardware is recommanded for this sensor. The default Esphome's logger also use one hardware UART.
+    The ESP8266 only has two hardware UARTs, one of which is transmit-only (details in `Logger Component section <components/logger>`__).
+
+
+So to avoid crashes you can choose between these 2 recommandations :
+
+- Disable the logging via UART :
+    
+.. code-block:: yaml
+
+    logger:     
+      baud_rate: 0   # disable logging via UART, help to avoid numerous crash with ESP_LOGD
+      level: INFO   # INFO for less log, put DEBUG to view all the linky's "étiquettes" received in the logs
+      esp8266_store_log_strings_in_flash: False     #  :doc:`recommanded for ESP8266 </components/sensor/custom>`
+
```

Comment:
I'm fine adding some examples on how to disable UART logging like you did but this is subject to changes so maybe it's better to simply reference the uart wiki page and let people figure out how to disable uart logging from there?

Reply:
Yes we could. But I think that the 2 examples can make gain a lot of time to the users. The part of the documentation about the uart is not the most simple to handle for beginners.